backend/api/Areas/Activities/Controllers/ActivityController.cs

```diff
@@ -76,6 +86,29 @@ public IActionResult GetActivityById(long activityId)
             return new JsonResult(_mapper.Map<ActivityInstanceModel>(activity));
         }
 
+        /// <summary>
+        /// Retrieves the activity with the specified id.
+        /// </summary>
+        /// <param name="fileType">The type of file the fileId belongs to.</param>
+        /// <param name="fileId">the id of the file.</param>
+        /// <returns></returns>
+        [HttpGet("{fileType}/file/{fileId:long}")]
```

Comment:
nit: the path looks a tad redundant 
```
[GET] activity/research/file/1
```
Could be instead
```
[GET] activity/researchFile/1
```
or just 
```
[GET] activity/research/1
```

Reply:
On that same vein, would it not be better to have the file controller handle that?
```
[GET] {fileType}/{fileId}/activities/

Reply:
I think our pattern has been, get {resource} by x where {resource} generally determines what controller the function should belong to (in this case, the activities controller).

Reply:
I think this also helps the file controller get out of control with its responsibilities, otherwise it would need to handle all documents, notes, activities, and other child relationships.